### PR TITLE
Test zypper info subcommand on src-pkg on SLE

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
WIP: This test needs the DVD2-SRC to be synched with https://gitlab.suse.de/openqa/scripts/blob/master/rsync.pl

https://progress.opensuse.org/issues/15932

### CODE REVIEW

**Comment by SergioAtSUSE** [comment]
Depends on: https://gitlab.suse.de/openqa/scripts/merge_requests/76

File: tests/console/zypper_info.pm [hunk 1]
```diff
@@ -20,30 +20,32 @@ use utils;
...
+      unless $info_output_vim =~ /$expected_package_name_vim/;
 
-    if (check_var('DISTRI', 'opensuse')) {
-        # sle not yet supported because of missing source repos
-        zypper_call('mr -e repo-source');
-        zypper_call('ref');
-        $info_output = script_output 'zypper info srcpackage:htop';
+    if (check_var('DISTRI', 'sle')) {
+        # use dvd2 as the src-repository
+        zypper_call('ar --disable --type plaindir cd:///?devices=/dev/sr1 repo-source');
```
Comment (coolo) [inline tests/console/zypper_info.pm]: hmm, your rsync changes sync a pool - not a DVD. This won't work

**Comment by okurz** [comment]
LGTM but need proper syncing beforehand

**Comment by coolo** [comment]
I think the syncing is fine - but the test needs to use the repo and not an ISO. Customers don't have a Source ISO

**Comment by SergioAtSUSE** [comment]
I was told that to create this pool, the ISO should be pulled first, and then I would have both available.
And I was also told that the customer receives both, binaries and sources: https://download.suse.com/Download?buildid=qb71v1YjQjI~

This first step should test the source DVD as a repo. But @sysrich should check if this is the right way to use the ISO.

After that it is planned in a near future to test with the pool of SCC too.

**Comment by coolo** [comment]
didn't know DVD2 is for download - but it makes sense actually for GPL reasons

**Comment by okurz** [comment]
source media directories are available now, e.g. ftp://openqa.suse.de/SLE-12-SP3-SERVER-POOL-x86_64-Build0295-Media2/

you can do something like https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/toolchain/crash.pm#L36 is doing for Media3 but for Media2 now. You can even test locally as you can access ftp://openqa.suse.de

**Comment by SergioAtSUSE** [comment]
Synching the iso: https://gitlab.suse.de/openqa/scripts/merge_requests/77

**Comment by SergioAtSUSE** [comment]
tested that the DVD2 with the sources of sles are properly synchronized in

osd:/var/lib/openqa/factory/iso/SLE-12-SP3-Server-DVD-x86_64-Build0300-Media2.iso

See here https://openqa.suse.de/tests/844467

**Comment by SergioAtSUSE** [comment]
edited the settings for the test suite "extra_tests_in_textmode"

adding this var

`ISO_2=%DISTRI%-%VERSION%-%ARCH%-%BUILD%-Media2.iso`

**Comment by SergioAtSUSE** [comment]
Tested on local instance, ready to be merged

http://copland.arch.suse.de/tests/176

**Comment by coolo** [comment]
I would appreciate if you guys wouldn't be soooo SP3 centric. Just because SP3 has a ISO_2 synced does not mean the test can run on all versions.

**Comment by okurz** [comment]
I guess you mean https://openqa.suse.de/tests/846850#step/zypper_info/10 ?

**Comment by okurz** [comment]
@SergioAtSUSE should handle https://progress.opensuse.org/issues/18204

**Pull request merged**